PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from pymoab import core, types, tag
+from sys import argv
+
+mb = core.Core()
+
+#Read the data file with the graveyard to be removed.
+mb.load_file(argv[1])
+
+tag_name = mb.tag_get_handle("NAME")
+tag_category = mb.tag_get_handle("CATEGORY")
+root = mb.get_root_set()
+
+#Retrieve all EntitySets with a Category tag with a value of "Group".
+group_categories = list(mb.get_entities_by_type_and_tag(root, types.MBENTITYSET,
+                                                        tag_category, np.array(["Group"])))
+
+#Retrieve all EntitySets with a Name tag.
+group_names = mb.tag_get_data(tag_name, group_categories, flat = True)
+
+#Find the EntitySet with the "graveyard" Name tag value.
+graveyard_sets = [group_set for group_set, name in zip(group_categories, group_names) 
+                  if name.lower() == b'graveyard']
+print(graveyard_sets)
```

Comment:
This print statement might be more useful for a user if it came with a comment about what is being printed.